requirements.txt

```diff
@@ -1,6 +1,5 @@
+--find-links hvac==https://github.com/mracter/hvac/archive/b0811f0446760125baefea6383711035104e6df1.zip#egg=hvac-0.2.17+git.b817da4
+hvac-0.2.17+git.b817da4
 setuptools==36.2.7
-hvac==0.2.17
-pytest==3.2.1
 requests==2.18.4
-responses==0.7.0
-python_daemon==2.1.2
+python_daemon==2.1.2
```

Comment:
You can remove this `requirements.txt` file now that the requirements are in `install_requires`...

Reply:
Removed redundant `requirements` files.